src/ts/components/uicontainer.ts

```diff
@@ -236,8 +243,6 @@ export class UIContainer extends Container<UIContainerConfig> {
     uimanager.getConfig().events.onUpdated.subscribe(() => {
       updateState(PlayerUtils.getState(player));
     });
-    // Init in current player state
-    updateState(PlayerUtils.getState(player));
```

Comment:
Why did you remove the initialization?

Reply:
The problem here is that the `updateState` now actually triggers a `showUI` before other components had a chance to subscribe to the `onControlsShow` callback. This happens if we are in a `hidingPreventedState` (Prepared). But within the `onShow` we have a flag (`isUiShown`) which will change into `true` but nothing is actual visible.

Call stack:
- `updateState`
- `stateChanged.dispatch`
- Calls `stateChanged.subscribe` callback in `uicontainer`
- `hidingPrevented` returns `true`
- calls `showUI`
- switches the `isUiShown` flag
- a second call would not trigger the `showUI` again

---

I think it's safe to remove the initialization here because the `uimanager` will fire `onUpdated` after initialization (if player is ready). Or the player will fire an appropriated event when it's ready.

Reply:
Makes sense 👍